app/Http/Controllers/SubscriberController.php

```diff
@@ -104,6 +104,15 @@ public function store(Request $request)
         $selectedLists[] = $allCategory->first()->id;
         $subscriber->lists()->attach($selectedLists);
 
+        if (isApi()) {
```

Comment:
what do you think about using wantsJson?
```suggestion
        if ($request->wantsJson()) {
```

https://laravel.com/docs/4.2/requests#request-information